[2025-04-22T22:00:42.931+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-22T22:00:42.938+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: BuildELT_dbt.dbt_test manual__2025-04-22T22:00:34.317889+00:00 [queued]>
[2025-04-22T22:00:42.941+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: BuildELT_dbt.dbt_test manual__2025-04-22T22:00:34.317889+00:00 [queued]>
[2025-04-22T22:00:42.941+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-22T22:00:42.945+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): dbt_test> on 2025-04-22 22:00:34.317889+00:00
[2025-04-22T22:00:42.949+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=696) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-22T22:00:42.948+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'BuildELT_dbt', 'dbt_test', 'manual__2025-04-22T22:00:34.317889+00:00', '--job-id', '174', '--raw', '--subdir', 'DAGS_FOLDER/build_elt_with_dbt.py', '--cfg-path', '/tmp/tmpgsool3sb']
[2025-04-22T22:00:42.950+0000] {standard_task_runner.py:72} INFO - Started process 697 to run task
[2025-04-22T22:00:42.950+0000] {standard_task_runner.py:105} INFO - Job 174: Subtask dbt_test
[2025-04-22T22:00:42.966+0000] {task_command.py:467} INFO - Running <TaskInstance: BuildELT_dbt.dbt_test manual__2025-04-22T22:00:34.317889+00:00 [running]> on host 30b8bbcaa7d3
[2025-04-22T22:00:42.994+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='BuildELT_dbt' AIRFLOW_CTX_TASK_ID='dbt_test' AIRFLOW_CTX_EXECUTION_DATE='2025-04-22T22:00:34.317889+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-22T22:00:34.317889+00:00'
[2025-04-22T22:00:42.994+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-22T22:00:43.000+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-22T22:00:43.001+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt test --profiles-dir /opt/***/dbt --project-dir /opt/***/dbt']
[2025-04-22T22:00:43.004+0000] {subprocess.py:86} INFO - Output:
[2025-04-22T22:00:44.193+0000] {subprocess.py:93} INFO - [0m22:00:44  Running with dbt=1.7.5
[2025-04-22T22:00:44.571+0000] {subprocess.py:93} INFO - [0m22:00:44  Registered adapter: snowflake=1.7.5
[2025-04-22T22:00:44.595+0000] {subprocess.py:93} INFO - [0m22:00:44  Found 3 models, 1 snapshot, 3 sources, 0 exposures, 0 metrics, 431 macros, 0 groups, 0 semantic models
[2025-04-22T22:00:44.596+0000] {subprocess.py:93} INFO - [0m22:00:44
[2025-04-22T22:00:44.596+0000] {subprocess.py:93} INFO - [0m22:00:44  Nothing to do. Try checking your model configs and model specification args
[2025-04-22T22:00:45.139+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-22T22:00:45.149+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-22T22:00:45.149+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=BuildELT_dbt, task_id=dbt_test, run_id=manual__2025-04-22T22:00:34.317889+00:00, execution_date=20250422T220034, start_date=20250422T220042, end_date=20250422T220045
[2025-04-22T22:00:45.190+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-22T22:00:45.199+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-22T22:00:45.200+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
